### PR TITLE
Article create env vars and env vars index

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

File: content/docs/environment-variables/creating-env-vars.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
----
-tags: example
----
-
 ## Creating environment variables
+
+The interface supports two types of data, this article will show how to create
+both a `text` and `SSH Key pair` evn var.
```
Comment (mies) [inline content/docs/environment-variables/creating-env-vars.md]: typo: env. I would also spell it out; environment variable (env var)

File: content/docs/environment-variables/creating-env-vars.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+To create an env var go to the `Pipeline` settings section of your application
+and click `Add new variable`.
+
+* [Creating a text env var](#text-env-var)
+* [Creating a SSH key pair env var](#ssh-env-var)
+
+### <a name="text-env-var"></a>Creating a text env var
+
+Creating a new env var is as simple as filling in a name, value and hit save.
+The next build or deploy you will trigger now has the env var available.
```
Comment (mies) [inline content/docs/environment-variables/creating-env-vars.md]: now has the env var available in its pipeline

File: content/docs/environment-variables/creating-env-vars.md [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+
+> Please not that the name has to start with a letter and after that the following characters are supported: 'a-z', '0-9' and '_'
```
Comment (mies) [inline content/docs/environment-variables/creating-env-vars.md]: the underscore is not correctly closed resulting in that everything below is italic

File: content/docs/environment-variables/creating-env-vars.md [hunk 4]
```diff
@@ -1,5 +1,65 @@
...
+
+* let wercker generate a pair, [read more on creating a SSH key pair &rsaquo;](/docs/ssh-keys/creating-ssh-keys.html)
+* create a variable
+
+#### Create a variable
+
+When you create a new variable for the `SSH key pair`, remember you are actually
+creating two variables who are based on the name you are entering.
+
+So, if you created an SSH key pair to use as a bitbucket deploy key. You may want to name
```
Comment (mies) [inline content/docs/environment-variables/creating-env-vars.md]: For instance, if you ...

Comment (mies) [inline content/docs/environment-variables/creating-env-vars.md]: I would combine these two sentences into one:

For instance if you created an SSH key pair to use as a bitbucket deploy key, you may want to name.........

File: content/docs/environment-variables/index.md [hunk 5]
```diff
@@ -1,14 +1,39 @@
...
+too sensitive to store in the repository. Wercker supports a number of ways to
+store and expose this data as environment variables.
 
-You can also use a [script step](/learn/steps/introduction.html) and use the export command to see the full list
-of all variables at that moment during the build/deploy.
+### Global env vars
 
-This is convenient as the environment variables step does not always show all
-environment variables available during the pipeline run.
+There are variables which are `global`. They are available during builds and deploys.
```
Comment (mies) [inline content/docs/environment-variables/index.md]: These are available

File: content/docs/environment-variables/index.md [hunk 6]
```diff
@@ -1,14 +1,39 @@
...
+Typically, these are used to store API tokens for after-steps such as hipchat, campfire etc.
+These variables are called pipeline variables and can be set in the settings tab
+of your application.
+
+[Read more on creating env vars &rsaquo;](/docs/environment-variables/creating-env-vars.html)
+
+### Deploy env vars
```
Comment (mies) [inline content/docs/environment-variables/index.md]: Spell out the title: Deploy environment variables

File: content/docs/environment-variables/index.md [hunk 7]
```diff
@@ -1,14 +1,39 @@
...
 
-## Using environment variables
+What if you have an API key you need during a deploy or during a build? This is
+information that is either unique for each server you want to deploy to or may be
+too sensitive to store in the repository. Wercker supports a number of ways to
+store and expose this data as environment variables.
 
-You can also use a [script step](/learn/steps/introduction.html) and use the export command to see the full list
-of all variables at that moment during the build/deploy.
+### Global env vars
```
Comment (mies) [inline content/docs/environment-variables/index.md]: Spell out the title: Global environment variables

**Comment by mies** [comment]
@lindseybateman minor comments. Note the closing of the underscore resulting everything after it being italic

**Comment by lindseybateman** [comment]
thanks

File: content/docs/environment-variables/creating-env-vars.md (hunk 1 above)
Comment (lindseybateman) [inline content/docs/environment-variables/creating-env-vars.md]: check

File: content/docs/environment-variables/creating-env-vars.md (hunk 2 above)
Comment (lindseybateman) [inline content/docs/environment-variables/creating-env-vars.md]: check

File: content/docs/environment-variables/creating-env-vars.md (hunk 3 above)
Comment (lindseybateman) [inline content/docs/environment-variables/creating-env-vars.md]: seems ok after rendering

File: content/docs/environment-variables/creating-env-vars.md (hunk 4 above)
Comment (lindseybateman) [inline content/docs/environment-variables/creating-env-vars.md]: check

File: content/docs/environment-variables/index.md (hunk 5 above)
Comment (lindseybateman) [inline content/docs/environment-variables/index.md]: check

File: content/docs/environment-variables/index.md (hunk 6 above)
Comment (lindseybateman) [inline content/docs/environment-variables/index.md]: check

File: content/docs/environment-variables/index.md (hunk 7 above)
Comment (lindseybateman) [inline content/docs/environment-variables/index.md]: check

**Pull request merged**